mailbag/formats/mbox.py

```diff
@@ -30,65 +30,95 @@ def account_data(self):
 
     def messages(self):
         
-        files = glob.glob(os.path.join(self.file, "**", "*.mbox"), recursive=True)
-        for filePath in files:
-            subFolder = helper.emailFolder(self.file,filePath)
+        if os.path.isfile(self.file):
+            files = self.file
+            parent_dir = os.path.dirname(self.file)
+        else:
+            files = os.path.join(self.file, "**", "*.mbox")
+            parent_dir = self.file
+        file_list = glob.glob(files, recursive=True)
+        for filePath in file_list:
+            subFolder = helper.emailFolder(parent_dir, filePath, True)
```

Comment:
TypeError: emailFolder() takes 2 positional arguments but 3 were given

Reply:
Fixed. This was left in from an alternate approach to `Message_Path`. Thanks for finding.